shared/css/color.scss

```diff
@@ -96,6 +96,11 @@ $level_review_accepted: rgb(11, 142, 11); // TODO: $level_passed;
 $assessment: $cyan;
 $workspace_running_background: #e5e5e5;
 
+// Sprite Lab validation colors.
+$spritelab_fail: #682A77;
+$spritelab_pass: #0ebe0e;
+$spritelab_bonus: #e9118f;
+
```

Comment:
I know there is some precedent with the progress colors, above, but do wonder whether this file should have app-specific information.  An alternative would be to have only colors in this file, and map from those colors to Sprite Lab-specific usages somewhere more specific to Sprite Lab.

Reply:
That makes sense to me! Take a look at the most recent commit for a new approach. I don't have time right now to re-work the entire color.scss file because I have no idea how many features depend on the current app-specific names. Should we take that on as future follow-up work?

Reply:
As discussed, we'll try for conventional naming and probably keep them separate until exact colours are settled upon.